.github/workflows/pr.yaml

```
@@ -22,6 +23,27 @@ jobs:
       - wheel-test-pylibwholegraph
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.12
+    if: always()
+    with:
+      needs: ${{ toJSON(needs) }}
+  changed-files:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@branch-24.12
+    with:
+      files_yaml: |
+        test_cpp:
+          - '**'
+          - '!.pre-commit-config.yaml'
+          - '!README.md'
+          - '!python/**'
+          # TODO: Remove this before merging
+          - '!.github/workflows/**'
+        test_python:
+          - '**'
+          - '!.pre-commit-config.yaml'
+          - '!README.md'
```

```suggestion
          - '!README.md'
          - '!scripts/checks/**'
```

As far as I can tell, nothing in `scripts/checks` is used by the Python tests. I think it'd be safe to skip those tests on PRs that only change those files.